### PR TITLE
Fix get for read instead of post

### PR DESCRIPTION
**Pull request by Sabfir**

### CODE REVIEW

**Comment by KostyaTk** [comment]
Reviewed

**Comment by Bmaks-github** [comment]
Reviewed

File: src/main/java/ua/softserve/rv_018/greentourism/controller/PlaceController.java [hunk 1]
```diff
@@ -31,52 +29,78 @@
...
+	 * The service returns the list of Point entities as JSON.
+	 * 
+	 * @return A ResponseEntity containing a List of Point objects.
+	 */
+	@RequestMapping(value = "/point", method = RequestMethod.GET,
+			headers = "Accept=application/json", produces = { "application/json" })
+	public ResponseEntity<?> getPlacePointsBetweenTwoCoordinates(
+			@RequestParam (value="south-west", required=true) String southWestParam,
+    		@RequestParam (value="north-east", required=true) String northEastParam) {
+		logger.info("> getPlacePointsBetweenTwoCoordinates");
```
Comment (Michaluch) [inline src/main/java/ua/softserve/rv_018/greentourism/controller/PlaceController.java]: use more informative message for logs

Comment (Sabfir) [reply src/main/java/ua/softserve/rv_018/greentourism/controller/PlaceController.java]: Log message changed

**Pull request merged**